### DISCUSSION TITLE
File with Video Stream

### DISCUSSION THREAD

**Discussion by AndrewDanyliuk** [zotify-dev/zotify#125] [Q&A] [answered]
I notice no matter what format I choose, zotify downloads the file with a video stream when checking with ffprobe.  It doesn't matter for my audio players, but my metadata scraper/fixer doesn't like it

Is there a setting to only save the audio stream? I've been running ffmpeg manually afterwards.

**Comment by AndrewDanyliuk** [comment, marked as answer]
Fixed this by modifying the outputs params in `convert_audio_format` of `track.py` to include `-vn`

**Reply by Hazeybxx** [reply]
Hey Andrew, are you able to help me out?
